ruby/spec/helpers/token.rb

```diff
@@ -0,0 +1,10 @@
+def create_token
+  creds = []
+  
+  File.foreach('config/.access') do |line|
+    creds.append(line.split('=', 2)[1].strip)
+  end
+
+  token = Base64.encode64("#{creds[0]}:#{creds[1]}")
```

Comment:
```suggestion
  content = File.read('config/.access')

  auth_string = content.split("\n").map{ |line| line.split('=').last }.join(':')

  Base64.encode64(auth_string)
```

WDYT?

Reply:
So these are pretty equivalent since that `Foreach` read the entire file first and then chops it up, just like you suggested. I just like the idea of an explicit FOR loop instead of MAP whenever possible.

I would only change it as follows:

```
creds = []
  
  File.foreach('config/.access') do |line|
    creds.append(line.split('=').last.strip)
  end
  
Base64.encode64(creds.join(':'))
```